edpm_ansible/roles/edpm_ovn/templates/ovn_metadata_agent.yaml.j2

```diff
@@ -0,0 +1,18 @@
+start_order: 2
+image: "{{ edpm_ovn_metadata_agent_image }}"
+net: host
+privileged: true
+user: root
+restart: always
+depends_on:
+  - openvswitch.service
+healthcheck:
+  test: '/openstack/healthcheck {{ edpm_ovn_sb_server_port }}'
+volumes:
```

Comment:
I guess we also need to add the TLS volumes in cases where TLS is enabled?
```
edpm_ovn_controller_tls_volumes
```

Like: https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/edpm_ansible/roles/edpm_ovn/templates/ovn_controller.yaml.j2#L19-L24

Reply:
Oh right, those volumes are for ovn_controller, my bad. So metadata agent doesn't care about TLS ever?

Reply:
Ok, seems fine. Just looked into the old issue we had in TripleO related to TLS and the metadata agent:
https://review.opendev.org/c/openstack/tripleo-heat-templates/+/605406/3/puppet/services/ovn-metadata.yaml


But it seems just changing these two variables for TLS-e deployments would be all we need to do:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/34/commits/c31541d6fa2e4650891be1a35f40241f7d0255b2#diff-61d75487470ab69a16e68b27db851584d6cb094c5a27e3dce29a9e8d2185ef4fR153-R154


All good, seems like we're covered for that scenario here then.